rewards-configuration.default.yml

```diff
@@ -0,0 +1,31 @@
+content-evaluator:
+  enabled: true
+user-extractor:
+  redeem-bounty: true
+data-purge:
+  enabled: true
+#disabled: true
+formatting-evaluator:
+  enabled: true
+  scores:
+    br: 1
+    code: 1
+    p: 0.1
+    em: 0
+    img: 0
+    strong: 0
+    blockquote: 0
+    h1: 1
+    h2: 1
+    h3: 1
+    h4: 1
+    h5: 1
+    h6: 1
+    a: 1
```

Comment:
The code puts them at 1 by default but I think you are right and the current configuration might reward 5 because I can't get the same results as the current bot with this config.